website/aws.erb

```diff
@@ -403,6 +403,15 @@
                     </ul>
                 </li>
 
+                <li<%= sidebar_current("docs-aws-resource-budget") %>>
+                    <a href="#">Budget Resources</a>
+                    <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-aws-resource-budget") %>>
```

Comment:
This should be updated to reflect the new nesting `sidebar_current("docs-aws-resource-budgets-budget")`